packages/@ama-sdk/core/src/plugins/reviver/reviver.spec.ts

```diff
@@ -45,7 +45,7 @@ describe('Reviver Reply Plugin', () => {
       response: {} as any
     });
 
-    const transformedData = await runner.transform(undefined);
+    const transformedData = await runner.transform();
```

Comment:
Not sure I am fond of this one if this is removing undefined parameters :/
I wonder if there is a risk for errors if we change the signature of the methods and add new parameters

Reply:
I hope it was able to remove it only because the parameter is optional.

Reply:
rule `@typescript-eslint/no-unsafe-assignment`